src/components/Stepper/js/StepItem.js

```diff
@@ -27,12 +21,41 @@ const StepItem = ({
         <div className="step-content">
           <h3 className="step-title">{step.props.title}</h3>
           <p className="step-description">{step.props.description}</p>
-          {isActive && React.cloneElement(step, { stepIndex: index })}
-          {stepError && <p className="error-message">{stepError}</p>}
         </div>
       </div>
+      <br />
+      <br />
+      {isCompleted ? (
+        <div className="progress-line" />
+      ) : (
+        <div className="progress-line-placeholder" />
+      )}
     </div>
   );
+
+  const renderVerticalStep = () => (
```

Comment:
There seems to be a bit too much duplicated code between `renderHorizontalStep` and `renderVerticalStep`. But let's say they are not that complicated and we can live with that.